### PR TITLE
Use osg-xrootd (SOFTWARE-3558)

### PR DESCRIPTION
**Pull request by brianhlin**
Most of the configuration has been moved to osg-xrootd: https://github.com/opensciencegrid/Software-Redhat/tree/trunk/osg-xrootd/osg

### CODE REVIEW

File: configs/stash-cache/config.d/50-stash-cache-paths.cfg [hunk 1]
```diff
@@ -8,9 +8,6 @@
 # This file is part of the StashCache Daemon
 # https://opensciencegrid.org/docs/data/stashcache/overview/
 
-# The directory on local disk where files will be cached, e.g. "/mnt/stash".
-oss.localroot $(rootdir)
```
Comment (matyasselmeci) [inline configs/stash-cache/config.d/50-stash-cache-paths.cfg]: Where will this get specified if you remove this line?

File: configs/stash-cache/xrootd/xrootd-stash-cache.cfg [hunk 2]
```diff
@@ -15,10 +15,12 @@
 # https://opensciencegrid.org/docs/data/stashcache/overview/
 #
 
-
 # A few extra configuration settings that have no good other location.
+set EnableHttp=1
```
Comment (matyasselmeci) [inline configs/stash-cache/xrootd/xrootd-stash-cache.cfg]: Where does this get used?

Comment (brianhlin) [reply configs/stash-cache/xrootd/xrootd-stash-cache.cfg]: https://github.com/opensciencegrid/Software-Redhat/blob/trunk/osg-xrootd/osg/50-osg-http.cfg

File: configs/stash-cache/config.d/50-stash-cache-paths.cfg (hunk 1 above)
Comment (brianhlin) [reply configs/stash-cache/config.d/50-stash-cache-paths.cfg]: https://github.com/opensciencegrid/Software-Redhat/blob/trunk/osg-xrootd/osg/50-osg-paths.cfg#L17-L18

**Pull request merged**